### ISSUE TITLE
Image updates in the dashboard

### ISSUE THREAD

**Issue by slesaad** [US-GHG-Center/veda-config-ghg#521]
# Description
Currently, we're using some placeholder images in various pages and stories and we're using the URL to the images.
However, since these are external images hosted by somebody else, we can't be sure that they'll always be hosted at the given URL/ they make take it down.

Thus, once the images are confirmed, we need to download the images, resize them and use them instead of the URLs.
Also, we need to make sure all the new images provide proper attribution to the owner/photographer.

In short:
- Download image
- Resize image (based on the instructions below 👇 )
- Switch remote image URL to local URL 
- Add attribution

Instructions for media: https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/media.md

# AC
- [x] All new images updated to use locally hosted images
- [x] All new images resized to reduce image size
- [x] All new images have attribution provided
